src/modules/form-dyn-messages.js

```diff
@@ -0,0 +1,78 @@
+(function ($) {
+
+	'use strict';
+
+	var html = $('html');
+
+	var sels = {
+		ctn: '.js-form-dyn-messages-ctn',
+		message: '.js-form-dyn-message',
+		template: '.js-form-dyn-message-template'
+	};
+
+	var resetMessage = function (msg) {
+		if (!!msg.length) {
+			App.modules.notify('changeState.update', {
+				item: msg,
+				state: 'visible',
+				action: 'off'
+			});
+			msg.empty();
+		}
+	};
+
+	var showMessage = function (form, fx) {
+		var ctn = form.find(sels.ctn);
+		var template = ctn.find(sels.template).filter('[data-fx="' + fx + '"]');
```

Comment:
The use of _fx_ in this context seems a bit counterintuitive to me, mostly for the user writing the xsl for a form. A first glance I would assume it would be a function, not a string. And on the markup side of things, `data-fx` could really mean anything. I'm probably being nitpicky here but I think that something like `data-msg-type="error"` or `data-message-for="error"` something a bit more verbose would be better. What do you think?

Reply:
@dominic-blain I totally agree. I like `data-message-for`. @PascalPiche @fhamon @beaubbe ?